config/default.schema.json

```diff
@@ -272,6 +272,11 @@
 				}
 			}
 		},
+		"WOPI_MAX_FILE_SIZE": {
+			"type": "integer",
+			"default": 20971520,
+			"description": "The maximum file size that can be handled by the WOPI API."
```

Comment:
Can you please add which data type file size represent in description or in name?
Like CONSENT_AGE_SECOND  or  PIN_MAX_AGE_SECONDS

Reply:
The description was updated.